dbt/schema.py

```diff
@@ -1,13 +1,46 @@
 from dbt.logger import GLOBAL_LOGGER as logger  # noqa
+import dbt.exceptions
+
+import google.cloud.bigquery
 
 
 class Column(object):
+    TYPE_LABELS = {
+        'STRING': 'TEXT',
+        'TIMESTAMP': 'TIMESTAMP',
+        'FLOAT': 'FLOAT',
+        'INTEGER': 'INT'
+    }
+
     def __init__(self, column, dtype, char_size=None, numeric_size=None):
         self.column = column
         self.dtype = dtype
         self.char_size = char_size
         self.numeric_size = numeric_size
 
+    @classmethod
+    def translate_type(cls, dtype):
+        return cls.TYPE_LABELS.get(dtype.upper(), dtype)
+
+    @classmethod
+    def create(cls, name, label=None, dtype=None):
```

Comment:
Based on how `translate_type` works, it looks like you could just pass one `label_or_dtype` argument and pass it to `translate_type` unconditionally, no need for two arguments.